### PR TITLE
Improving the test

### PR DESCRIPTION
**Pull request by manuelam20**

### CODE REVIEW

File: protractor/local.config.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
 
 export const config: Config = {
   framework: 'jasmine',
   specs: [ '../test/google.spec.js' ],
-  seleniumAddress: 'http://localhost:4444/wd/hub'
+  seleniumAddress: 'http://localhost:4444/wd/hub',
+  onPrepare: () => {
+    browser.ignoreSynchronization = true;
+  }
 };
```
Comment (jhenaoz) [inline protractor/local.config.ts]: Agregar fin de linea

File: test/google.spec.ts [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+  describe('when open Google Page', () => {
+    beforeEach(() => {
+      browser.get('http://www.google.com');
+    });
+
+    it('then should have a title', () => {
+      expect(browser.getTitle()).toEqual('Google');
+    });
   });
 });
```
Comment (jhenaoz) [inline test/google.spec.ts]: agregar fin de linea

**Review by jhenaoz — COMMENTED** [review]
Just a couple of comments

File: protractor/local.config.ts (hunk 1 above)
Comment (manuelam20) [reply protractor/local.config.ts]: added

File: test/google.spec.ts (hunk 2 above)
Comment (manuelam20) [reply test/google.spec.ts]: added

**Review by jhenaoz — APPROVED** [review]

**Pull request merged**